packages/hagrid/hagrid/cli.py

```diff
@@ -21,6 +22,9 @@
 import click
 import rich
 
+# syft absolute
+import syft
```

Comment:
I'm not sure this is a good idea to have syft imported directly into HAGrid. Even when we add the ability for HAGrid to manage syft environments for us we probably won't want to import syft itself.

Reply:
@madhavajay, right also since we are dumping all other stuff on the `debug` command, I'll strip it off.